sdks/java/nexmark/src/test/java/org/apache/beam/sdk/nexmark/PerfsToBigQueryTest.java

```diff
@@ -92,8 +92,11 @@ public void testSavePerfsToBigQuery() throws IOException, InterruptedException {
     HashMap<NexmarkConfiguration, NexmarkPerf> perfs = new HashMap<>(2);
     perfs.put(nexmarkConfiguration1, nexmarkPerf1);
     perfs.put(nexmarkConfiguration2, nexmarkPerf2);
-    Instant start = Instant.now();
-    Main.savePerfsToBigQuery(options, perfs, fakeBqServices, start);
+    //cast to int due to BEAM-4734. To avoid overflow on int capacity,
```

Comment:
Nit: please add blank line between line 94 and 95.
Nit2: add space before "cast"

Let's drop the ": Beam incubation date :)" part (imo, it may be distractive)

Reply:
+1